src/render-html/webViewEventHandlers.js

```diff
@@ -92,6 +112,44 @@ export const handleImage = (props: Props, event: MessageListEventImage) => {
   }
 };
 
+export const handleLongPress = (
```

Comment:
Why are you not redirecting this to `MessageListContainer`. You don't need to recreate it!

Reply:
Moved to `MessageListContainer`. Now both `render-html` & `render-native` shares same code.